client/engine/engine.go

```diff
@@ -253,6 +253,8 @@ func (e *Engine) handleAPIEvent(apiEvent APIEvent) (ObjectiveChangeEvent, error)
 			if err != nil {
 				return ObjectiveChangeEvent{}, fmt.Errorf("handleAPIEvent: Could not create objective for %+v: %w", request, err)
 			}
+			// If ddfo creation was successful, destroy the consensus channel to prevent it being used (a Channel will now take over governance)
+			e.store.DestroyConsensusChannel(request.ChannelId)
```

Comment:
Unfortunately, there will be a nonzero amount of time between destroying the `ConsensusChannel`, and writing the `Channel` to the store. We can improve that here by using a `defer` statement...